webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/ShowNodeDetailsFromNode.scala

```diff
@@ -305,7 +310,11 @@ class ShowNodeDetailsFromNode(
     "#node_parameters -*" #> agentPolicyModeEditForm.cfagentPolicyModeConfiguration(Some(node.id)) &
     "#node_parameters -*" #> (if (node.rudderSettings.isPolicyServer) NodeSeq.Empty
                               else agentScheduleEditForm(node).cfagentScheduleConfiguration) &
-    "#node_tabs [id]" #> s"details_${id}").apply(serverDetailsTemplate)
+    "#node_tabs [id]" #> s"details_${id}" &
+    "href=#node_summary -*" #> <span class={s"badge-compliance-score ${globalScore.value.value} sm"}></span> &
+    "href=#node_reports -*" #> <span class={
+      s"badge-compliance-score ${globalScore.details.find(_.scoreId == "compliance").map(_.value).getOrElse(NoScore).value} sm"
```

Comment:
Can you declare `compliance` as the ID of the compliance score, it's a special keyword for scores, we need to have a convention for them. 
Perhaps in the handler, it looks like the only place where we give an identity to a compliance score: 
```
object ComplianceScoreEventHandler extends ScoreEventHandler {
  implicit val compliancePercentEncoder: JsonEncoder[ComplianceSerializable]     = DeriveJsonEncoder.gen

  val id = "compliance"

  def handle(event: ScoreEvent):         PureResult[List[(NodeId, List[Score])]] = {
    event match {
      case ComplianceScoreEvent(n, percent) =>
```

Or have a `ComplianceScoreId` object somewhere ?